app/resources/en/translation/dashboard.json

```diff
@@ -8,7 +8,7 @@
   "tableHeaderActions": "Actions",
   "statusDraftProbate": "Draft (probate)",
   "statusDraftIntestacy": "Draft (intestacy)",
-  "statusInProgress": "In progress",
+  "statusInProgress": "IN PROGRESS",
```

Comment:
I cant find this capitalisation on the JIRAs, but assuming its correct for the dashboard